istio/istio.go

```diff
@@ -383,7 +384,34 @@ func (iClient *IstioClient) executeBookInfoInstall(ctx context.Context, arReq *m
 	return nil
 }
 
-// ApplyRule is a method invoked to apply a particular operation on the mesh in a namespace
+func (iClient *IstioClient)executeHipsterShopInstall(ctx context.Context, arReq *meshes.ApplyRuleRequest) error {
+
+	resp, err := http.Get("https://raw.githubusercontent.com/GoogleCloudPlatform/microservices-demo/master/release/istio-manifests.yaml")
+	if err != nil {
+		panic(err)
+	}
+	defer resp.Body.Close()
+	body, err := ioutil.ReadAll(resp.Body)
+
+	if err := ioutil.WriteFile("merged.yaml", body, 0644); err != nil {
```

Comment:
instead of writing to file twice, can we just store this in a variable here? We can call it `yamlFileContents`.